deployer/templates/fluentd.yaml

```diff
@@ -150,6 +150,12 @@ objects:
               value: ${JOURNAL_SOURCE}
             - name: "JOURNAL_READ_FROM_HEAD"
               value: ${JOURNAL_READ_FROM_HEAD}
+            - name: "USE_MUX"
```

Comment:
They are purely for the MUX test script.  The script redeploys logging-fluentd with its daemonset patched.  Once logging-fluentd restarted, the original MUX environment variables are wiped out and there was no way to reset them in the script unless having them in the template file...